### PR TITLE
card design added

### PR DESCRIPTION
**Pull request by hangingPandaReact**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -2,15 +2,12 @@ import {} from 'react-native';
...
-import Card from './src/components/card/Card';
-import Header from './src/components/Header';
+import PostScreen from './src/screen/postScreen/PostScreen';
 
 const App = () => {
   return (
     <Provider store={store}>
-      <Card />
-      <Header />
+      <PostScreen />
```
Comment (hangingPandaReact) [inline App.js:10]: Navigation setup should be done

File: src/components/Header.js [hunk 2]
```diff
@@ -9,7 +8,7 @@ const Header = () => {
   return (
     <View style={styles.container}>
       <View>
-        <AntDesign name="home" size={25} color={colors.white} />
+        <Ionicons name="menu-outline" size={25} color={colors.white} />
```
Comment (hangingPandaReact) [inline src/components/Header.js:11]: Component should be created for this

File: src/movies/Movies.json [hunk 3]
```diff
@@ -90,7 +90,7 @@
     "Writer": "Mark Fergus (screenplay), Hawk Ostby (screenplay), Art Marcum (screenplay), Matt Holloway (screenplay), Stan Lee (characters), Don Heck (characters), Larry Lieber (characters), Jack Kirby (characters)",
     "Actors": "Robert Downey Jr., Terrence Howard, Jeff Bridges, Gwyneth Paltrow",
     "Plot": "After being held captive in an Afghan cave, billionaire engineer Tony Stark creates a unique weaponized suit of armor to fight evil.",
-    "Language": "Hungarian, Kurdish, Hindi, English, Persian, Urdu, Arabic",
+    "Language": "Hindi, English",
```
Comment (hangingPandaReact) [inline src/movies/Movies.json:93]: Remove this Json once you implement the API call

**Comment by deepakhangingpanda** [comment]
completed

**Pull request merged**